RoddyCore/test/src/de/dkfz/roddy/execution/io/fs/BashCommandSetSpecification.groovy

```diff
@@ -25,18 +25,36 @@ class BashCommandSetSpecification extends Specification {
     @Shared
     File tmpc = new File("/tmp/c")
 
+    def "test getListFilesInDirectoryCommand without filters"(File dir, expected) {
+        expect:
+        b.getListFilesInDirectoryCommand(dir) == expected
+
+        where:
+        dir  | expected
+        tmpa | 'find -L /tmp/a -type f -maxdepth 1'
+    }
+
+    def "test getListFilesInDirectoryCommand with filters"(File dir, List<String> filters, expected) {
+        expect:
+        b.getListFilesInDirectoryCommand(dir, filters) == expected
+
+        where:
+        dir  | filters             | expected
+        tmpa | ["a", "??b", "*.c"] | 'ls -lL -d /tmp/a/a /tmp/a/??b /tmp/a/*.c 2> /dev/null | grep -v "^d" | awk \'{ print $9 }\' | uniq'
+    }
+
     def testGetListFullDirectoryContentRecursivelyCommand(File directory, int depth, FileType selectedType, boolean complexList, String result) {
         expect:
         b.getListFullDirectoryContentRecursivelyCommand(directory, depth, selectedType, complexList) == result
 
         where:
         directory | depth | selectedType | complexList | result
-        tmpa      | -1    | ANY          | false       | 'find "/tmp/a"'
-        tmpa      | -1    | FILES        | false       | 'find "/tmp/a" -type f'
-        tmpa      | -1    | DIRECTORIES  | true        | 'find "/tmp/a" -type d -ls'
-        tmpa      | 0     | ANY          | false       | 'find "/tmp/a"'
-        tmpa      | 1     | FILES        | false       | 'find "/tmp/a" -maxdepth 1 -type f'
-        tmpa      | 2     | DIRECTORIES  | true        | 'find "/tmp/a" -maxdepth 2 -type d -ls'
+        tmpa      | -1    | ANY          | false       | 'find -L "/tmp/a"'
```

Comment:
The test fails for my build because the string is built differently in [Lines 200-208 of `BashCommandSet`](https://github.com/TheRoddyWMS/Roddy/pull/315/files#diff-749a2fe9bb0f19fdc8ef2823b54443bdR203)

- `-L` is appended after the path AND only if depth > 0